conda/environments/cuml_dev_cuda10.0.yml

```diff
@@ -37,6 +37,7 @@ dependencies:
 - recommonmark
 - pip
 - pip:
+    - ipython
```

Comment:
Could you expand on why ipython is necessary now? Don't get me wrong, I like ipython, but it's nice to keep the dev env small if possible.
If we need it, would be good to do conda instead if possible.

Reply:
ah, looks like we do need it for docs build.

Reply:
Yep.